### PR TITLE
Add UFC Fight Pass

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/u/ufcfightpass.com.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "UFC Fight Pass": {
+    "domain": "ufcfightpass.com",
+    "contact": {
+      "facebook": "UFCFightPass",
+      "twitter": "UFCFightPass"
+    },
+    "categories": ["entertainment"]
```
Comment (hkamran80) [inline entries/u/ufcfightpass.com.json]:
```suggestion
    "categories": [
      "entertainment"
    ]
```

File: img/u/ufcfightpass.com.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</g>
+</svg>
```
Comment (hkamran80) [inline img/u/ufcfightpass.com.svg]: Here's an SVG that's just the UFC logo.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1675 580"><path fill="#d20a0a" d="M136 0h199L216 426c-1 5-1 11 2 15 5 8 14 11 23 11h31c11 0 23 1 33-3 8-3 12-10 14-18L439 0h198L511 453l-14 48c-6 20-16 39-32 52a99 99 0 0 1-41 19c-21 5-42 7-64 8H104c-25-2-51-7-72-22-14-9-25-23-29-39-3-7-2-15-2-22 0-9 3-17 5-25L137 0h-1Zm595 0h427l-36 127H695L731 0Zm619 0h324l-15 57-20 70h-210c-10 1-21 0-31 4-7 4-12 12-14 20l-76 276c-2 6-1 13 4 17 6 6 15 8 24 8h229l-35 128h-325c-24-2-49-7-70-20-13-9-24-22-30-37-5-13-4-28-1-41l110-396c6-20 14-39 29-54 9-9 21-16 34-20 24-8 49-10 73-12ZM671 215h427l-36 128H833l-66 237h1-198l101-365Z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Made the changes.

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you for your contributions!

**Pull request merged**